### PR TITLE
SIM: MPC: add two x500 lidar airframes

### PR DESCRIPTION
**Pull request by Claudio-Chies**
Partly Tracked in https://github.com/PX4/PX4-Autopilot/issues/23602
To be merged only when following PR is merged
- [x] https://github.com/PX4/PX4-gazebo-models/pull/62 

**other relevant PR's**
- [ ] https://github.com/PX4/PX4-user_guide/pull/3452

### Overview
This PR Adds two X500 models, one with a front facing rangefinder for collision prevention, and one down facing lidar for normal rangefinder use case testing.
I also renamed the x500_lidar to x500_lidar_2d, to better differentiate the three

### Changes:
**New Models:**
`x500_lidar_front`
`x500_lidar_front`
**Changes:**
`x500_lidar `--> `x500_lidar_2d`

### Changelog Entry
For release notes:
```
Feature: Added front and downfacing MPC Models for SITL Testing
Documentation: read docs.px4.io/main/en/sim_gazebo_gz/vehicles.html
```

### Test coverage
- [x] Tested in SITL

### Open Tasks:
- [x] Update the Docs 
- [x] Update the Git Submodules, once the models have been merged.
- [x] Add Meshes to the sensors.

### CODE REVIEW

File: src/modules/simulation/gz_bridge/GZBridge.cpp [hunk 1]
```diff
@@ -762,6 +770,44 @@ void GZBridge::navSatCallback(const gz::msgs::NavSat &nav_sat)
...
 }
+void GZBridge::laserScantoLidarSensorCallback(const gz::msgs::LaserScan &scan)
+{
+	if (hrt_absolute_time() == 0) {
+		return;
+	}
+
+	distance_sensor_s distance_sensor{};
+	distance_sensor.timestamp = hrt_absolute_time();
+	distance_sensor.device_id = 0;
```
Comment (dakejahl) [inline src/modules/simulation/gz_bridge/GZBridge.cpp:781]:
```suggestion
	device::Device::DeviceId id;
	id.devid_s.bus_type = device::Device::DeviceBusType::DeviceBusType_SIMULATION;
	id.devid_s.bus = 0;
	id.devid_s.address = 0;
	id.devid_s.devtype = DRV_DIST_DEVTYPE_SIM;
	distance_sensor.device_id = id.devid;
```

File: src/modules/simulation/gz_bridge/GZBridge.hpp [hunk 2]
```diff
@@ -49,6 +49,7 @@
 #include <uORB/SubscriptionInterval.hpp>
 #include <uORB/topics/parameter_update.h>
 #include <uORB/topics/differential_pressure.h>
+#include <uORB/topics/distance_sensor.h>
```
Comment (dakejahl) [inline src/modules/simulation/gz_bridge/GZBridge.hpp:52]:
```suggestion
#include <uORB/topics/distance_sensor.h>
#include <lib/drivers/device/Device.hpp>
```

**Comment by hamishwillee** [comment]
For docs, we want to know how they are used/invoked, what they allow you to test when using that frame (such as terrain following, collision detection etc) [i.e. if enabled by default], and any limits - i.e. rangefinder max/min distance.

**Comment by Claudio-Chies** [comment]
@Jaeyoung-Lim yea. This should be possible, as the two publish to different topics, and GZBridge is able to differentiate them by that.

**Pull request merged**